lib/Model.php

```diff
@@ -1770,24 +1770,30 @@ public static function exists(mixed $conditions = []): bool
 
     /**
      * @see Relation::take()
+     *
+     * @return Model|array<Model>|null
```

Comment:
Let's not duplicate documentation. That's what the @see link is for, and in any case `Model` is not correct (`static` is correct)

Same comment for `first` and `last`

Reply:
Eh, don't listen to me, you were on the right track and adding return type to these is a good idea; I just tried it, and while it doesn't make any difference to the static checker (because we have the extensions), it does improve the IDE intellisense functionality. 

I went ahead and added them in a separate commit: https://github.com/php-activerecord/activerecord/commit/8d25f06fe290c4707e30f5f63c21441530ecac32